rimage/file_simple.c

```diff
@@ -13,10 +13,10 @@
 
 #define BYT_IRAM_BASE		0xff2c0000
 #define BYT_IRAM_HOST_OFFSET	0x0C0000
-#define BYT_IRAM_SIZE		(80 * 1024)
+#define BYT_IRAM_SIZE		(8 * 32 * 1024)
```

Comment:
Probably worth double checking this, it maybe that the address space is 256kB but only 80kB is populated ? The docs I have all showed the smaller size.

Reply:
yes, there's only 80 KB, why would you want to change this setting for an unrelated GCC compilation issue on a different platform anyways?

Reply:
I just double-checked the HAS, it's 80 KB IRAM, 160 KB Data RAM, 48 KB I$, 96 KB D$

No idea how the 256 KB reference came from.